cdap-unit-test/src/test/java/co/cask/cdap/test/app/ServiceLifeCycleTestRun.java

```diff
@@ -82,11 +83,12 @@ public static void init() throws IOException {
     artifactJar = createArtifactJar(ServiceLifecycleApp.class);
   }
 
+  // TODO: Due to changes in netty executor, this test doesn't apply. Need to rework on it.
```

Comment:
File a JIRA, so we remember to come back to it?

Reply:
Filed CDAP-12765